components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@elasticpath/store-components",
-  "version": "1.1.9",
+  "version": "1.1.10",
```

Comment:
Please rebase an increment as we've had to update this since the last review

Reply:
Will do